images/external-dns/tests/02-e2e.sh

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env bash
+
+# monopod:tag:k8s
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+set +t
+
+function preflight() {
+	if [[ "${IMAGE_REGISTRY}" == "" ]]; then
```

Comment:
@joshrwolf these variables don't seem to be used, is this test just testing the upstream installation without using our images? 🤔